### PR TITLE
W-13857703 Add info from debugging Google doc

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by JaatadiaMulesoft — APPROVED** [review]

File: gateway/1.5/modules/ROOT/pages/policies-pdk-debug-deployed-policies.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+* <<debug-environment>>
+* <<debug-logs>>
+
+[[debug-environment]]
+== Use the PDK Debugging Environment
+
+The PDK debugging environment provides an easy way to view Flex Gateway logging and send a variety of requests. However, there may be some differences between it and your production environment. Use the PDK debugging environment before debugging on a Flex Gateway instance.
+
+For more information about PDK's debugging environment, see xref:policies-pdk-debug-local.adoc[]
```
Comment (sarathecoubian) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-debug-deployed-policies.adoc]:
```suggestion
For more information about PDK's debugging environment, see xref:policies-pdk-debug-local.adoc[].
```

File: gateway/1.5/modules/ROOT/pages/policies-pdk-debug-local.adoc [hunk 2]
```diff
@@ -146,3 +145,17 @@ curl --location --request POST 'http://0.0.0.0:8081/some/route' \
...
+
+By default, the PDK debugging environment enables logs with the `debug` loglevel configured.
+
+To change the loglevel, edit the `logging.runtimeLogs.logLevel` value in `test/config/logging.yaml`.
+
+To learn more about editing a Flex Gateway's logging configuration file, see xref:flex-local-third-party-logs-config.adoc[].
+
+== Use the Rust Debugger
+
+Visual Studio Code (VS Code) includes a Rust debugger that can run unit tests. You cannot use the VS Code Debugger while the policy is deployed on a Flex Gateway. Meaning that to create the API mocking, you must use unit tests for the specific logic you need to debug.
```
Comment (sarathecoubian) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-debug-local.adoc]: Nitpick: "Meaning" in this context might not translate properly. I don't think you necessarily need a transition between those two sentences.

File: gateway/1.5/modules/ROOT/pages/policies-pdk-debug-local.adoc [hunk 3]
```diff
@@ -146,3 +145,17 @@ curl --location --request POST 'http://0.0.0.0:8081/some/route' \
...
+
+To debug Rust code in VS Code, you must install the https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb[CodeLLDB Debugger Extension ^] or a different VS Code extension with the same functionality. To learn more about debugging Rust in VS Code, see https://code.visualstudio.com/docs/languages/rust#_debugging[Rust in Visual Studio Code ^].
```
Comment (sarathecoubian) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-debug-local.adoc]: Rich text preview shows a space between the final link and the period at the end of that sentence. I'd test (if you haven't already) to make sure the xref preview text and carets actually need spaces between them.

**Review by sarathecoubian — APPROVED** [review]

File: gateway/1.5/modules/ROOT/pages/policies-pdk-debug-local.adoc (hunk 2 above)
Comment (glenn-rodgers-sf) [reply gateway/1.5/modules/ROOT/pages/policies-pdk-debug-local.adoc]: Definitely don't need it!

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**